tests/integration/targets/inventory_aws_mq/playbooks/tasks/mq_instance_delete_cmd.yml

```diff
@@ -0,0 +1,17 @@
+---
+- include_tasks: find_broker.yml
+- name: remove broker instance
+  command: >
+    aws mq delete-broker --broker-id {{ broker_id }}
```

Comment:
Why don't you use the Ansible mq_broker module to delete the broker https://github.com/ansible-collections/community.aws/blob/main/plugins/modules/mq_broker.py?

Reply:
I thought it would be clearer to have the tests follow the same strategy (command or mq_broker module), and since I was not able to use the mq_broker, I left it as is.

I'm happy to switch the test to use the mq_broker module if you're ok to drop the keyed group integration test.

Reply:
I'm giving a look at the mq_broker module. I let you know.

Reply:
At a first sight, I can see that mq_broker_info returns always the tags: {} parameter. Where applicable you could replace cue cli commands with the mq_broker_info module.

You can also use the mq_broker to delete the broker. It should not impact the testing.

Still looking for mq_broker.

Reply:
I can confirm that tagging does not work on mq_broker. I'm going to open an issue for this. So, you can use the cli when you create the broker and just add a comment that you did that because tagging does not work in the mq_broker module.

Reply:
thanks, was planning on doing that after finishing with this one :+1: 
comment added.

Reply:
@alialkhalidi I guess you can delete the broker using the mq_broker module without any issue. The issue happens when you create a new broker because it does not get tagged, but otherwise it should work.

Reply:
I guess something like this should work:

```
- name: delete a broker
   community.aws.mq_broker:
     broker_name: "{{ broker_name }}"
     state: absent
```